source/channels.rst

```diff
@@ -50,3 +73,36 @@ or HTML to that repository (markdown preferred).
 In particular we highly encourage Dask users to contribute stories of
 how they have used Dask to solve interesting problems.
 
+
+Youtube
+-------
+
+The `Dask Youtube channel
+<https://www.youtube.com/channel/UCj9eavqmvwaCyKhIlu2GaoA?view_as=subscriber>`_
+holds content that we generate as part of our documentation.
+We also maintain `playlists to other Dask talks
+<https://www.youtube.com/channel/UCj9eavqmvwaCyKhIlu2GaoA/playlists?view_as=subscriber>`_
+
+For Dask contributors looking to produce screencast content we encourage the
+following tips:
+
+1.  Get a good microphone.  Headset microphones often cost around $30, and have
```

Comment:
This is good information, thank you.
You might still want to produce a screencast on how to make a technical screencast - I think most OSS devs like myself are pretty reticent to try it.

Reply:
This turns out to be technologically hard.  The screencast software (quicktime) doesn't want to run twice.

It's *super easy* though.  Start QuickTime.  Select Screen Recording from the File menu.  Click the red button.  Start talking.